SeaPublicWebsite.UnitTests/QuestionFlowServiceTests.cs

```diff
@@ -1365,36 +1404,73 @@ public void RunSkipLinkTestCases(QuestionFlowServiceTestCase testCase)
                 new { reference = "ABCDEFGH" }
             )),
         new(
-            "Accessible loft space continues to roof insulated if user has it",
+            "Loft space continues to accessible loft if user has it",
+            new Input(
+                QuestionFlowPage.LoftSpace,
+                "ABCDEFGH",
+                loftSpace: LoftSpace.Yes
+            ),
+            new PathByActionArguments(
+                nameof(EnergyEfficiencyController.AccessibleLoft_Get),
+                "EnergyEfficiency",
+                new { reference = "ABCDEFGH" }
+            )),
+        new(
```

Comment:
What about tests for "Loft space continues to accessible loft if user does not have loft space"